revops/index.html

```diff
@@ -56,15 +61,30 @@ <h2 class="section-heading">Our Expertise</h2>
       </div>
     </div>
     <div class="collection-list-wrapper w-dyn-list">
-      <div class="collection-list w-dyn-items" id="services"></div>
+      <div class="collection-list w-dyn-items">
+        {% for s in page.services %} {% for service in site.services %} {% if s
+        == service.slug %}
+        <div class="collection-item w-dyn-item">
+          <a href="{{service.url}}" class="link-block w-inline-block">
```

Comment:
Until we have more content on the services we should remove the link imo.

We can launch with just the excerpts here, but without showing the main pages.

Reply:
Already addressed in: https://github.com/derangeddk/deranged.dk/pull/97